source/accessibility.html.erb

```diff
@@ -55,7 +55,7 @@ title: Accessibility – GOV.UK Verify
           How accessible this website is
         </h2>
         <p class="govuk-body">
-          We are not aware of any current accessibility issues with this website.
+          We know some parts of this website are not fully accessible. There are parts of this website that are not accessible because of issues caused by the Verify status page.
```

Comment:
CHANGE
caused by the Verify status page.

TO
caused by the GOV.UK Verify status page.

REASON
The full name of the product is 'GOV.UK Verify'. We call it 'Verify' for shorthand within GDS.